### ISSUE TITLE
Train code

### ISSUE THREAD

**Issue by 123res** [sunwei925/CompressedVQA-HDR#2]
I greatly appreciate your excellent work. May I ask if you will be uploading the **train** code?

**Comment by sunwei925** [comment]
Thank you for your interest in our project. We do not plan to release the training code. However, you can still reproduce the training process by referring to our previous projects and simply replacing the model, training dataset, and loss function. Specifically, for the FR code, you may refer to [CompressedVQA](https://github.com/sunwei925/CompressedVQA), and for the NR code, you may refer to [StairIQA](https://github.com/sunwei925/StairIQA). We built our training code based on these two projects.
